src/coreclr/vm/appdomain.hpp

```diff
@@ -1010,6 +1010,13 @@ class BaseDomain
         return ::CreateHandle(m_handleStore, object);
     }
 
+    void DestroyHandle(OBJECTHANDLE handle)
```

Comment:
Is this needed? We call the static DestroyHandle directly everywhere else in the VM.